example_configs/apacheguacamole.md

```diff
@@ -0,0 +1,40 @@
+# Configuration for Apache Guacamole
+!! IMPORTANT - LDAP only works with LLDAP if using a [database authentication](https://guacamole.apache.org/doc/gug/ldap-auth.html#associating-ldap-with-a-database).  The Apache Guacamole does support using LDAP to store user config but that is not in scope here.
+This was achieved by using the docker [jasonbean/guacamole](https://registry.hub.docker.com/r/jasonbean/guacamole/).
+
+##  To setup LDAP
+
+Open and edit your Apache Guacamole properties files
+
+Located at `guacamole/guacamole.properties`
+
+Uncomment and insert the below into your properties file
+
+```
+### http://guacamole.apache.org/doc/gug/ldap-auth.html
+### LDAP Properties
+ldap-hostname: localhost
+ldap-port: 3890
+ldap-user-base-dn: ou=people,dc=example,dc=com
+ldap-username-attribute: uid
+ldap-search-bind-dn: uid=admin,ou=people,dc=example,dc=com
+ldap-search-bind-password: replacewithyoursecret
+ldap-user-search-filter: (memberof=cn=lldap_apacheguac,ou=groups,dc=example,dc=com)
+```
+
+*Exclude `ldap-user-search-filter` if you do not want to limit users based on a group(s)
```

Comment:
Nit: this markdown doesn't render properly, you need a space after the *
You can click on the three dots in the top-right corner of the file and "View the file" to see the markdown rendered.

Reply:
Thank you, I was actually going for one of those * wildcard caveats that you see when reading a passage or nutrition facts. 

I changed it to bullets as you suggested.

Reply:
I don't see the change, are you sure you made it? Did you commit?